src/wallet/wallet.ts

```diff
@@ -142,6 +145,11 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
 
     // Setup the connection so clients can listen to its events before it is started
     this.conn = new WalletServiceConnection();
+    if (enableWs) {
+      this._isWsEnabled = true;
+    } else {
+      this._isWsEnabled = false;
+    }
```

Comment:
Can't we do something like:

```suggestion
    this._isWsEnabled = enableWs;
```

Since `enableWs` has a default value on the constructor?